QA_Requests/PerfTickets/testCases/conftest.py

```diff
@@ -0,0 +1,120 @@
+import os
+from configparser import ConfigParser
+from pathlib import Path
+
+import pytest
+from QA_Requests.PerfTickets.userInputs.user_inputs import UserData
+from QA_Requests.PerfTickets.testPages.base.login_page import LoginPage
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+
+global driver
+
+
+@pytest.fixture(scope="session")
+def environment_settings():
+    """Load settings from os.environ
+
+            Names of environment variables:
+                DIMAGIQA_URL
+                DIMAGIQA_LOGIN_USERNAME
+                DIMAGIQA_LOGIN_PASSWORD
+                DIMAGIQA_MAIL_USERNAME
+                DIMAGIQA_MAIL_PASSWORD
+
```

Comment:
we can modify these env variables according to ones used in the settings.cfg

Reply:
I removed env variables totally because they won't be used here
https://github.com/dimagi/dimagi-qa/pull/137/commits/8fa03009d5d9714d8a4724bb34c4ee738d0710dc

Reply:
does it mean that the ones present in the settings-sample.cfg is also not needed?
`login_username = 
login_password = 
auth_key = `

Reply:
It means only those will be used. 
Since we won't run this on CI, I removed env variables altogether.